sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -103,12 +103,12 @@
     c.client_net_address as client_address,
     sess.host_name as host_name,
     sess.program_name as program_name,
-	dbspu.user_objects_alloc_page_count as user_objects_alloc_page_count,
+    dbspu.user_objects_alloc_page_count as user_objects_alloc_page_count,
     dbspu.user_objects_dealloc_page_count as user_objects_dealloc_page_count,
     dbspu.internal_objects_alloc_page_count as internal_objects_alloc_page_count,
     dbspu.internal_objects_dealloc_page_count as internal_objects_dealloc_page_count
 FROM sys.dm_exec_sessions sess
-	INNER JOIN sys.dm_db_session_space_usage dbspu
```

Comment:
On thing to consider when interpreting the metrics is that, according to [doc](https://learn.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-db-session-space-usage-transact-sql?view=sql-server-ver16), the performance counters are updated when a task completes. So if we have an active long runner, the space usage metric can be zero, even though the space is allocated.